backend/code_review_backend/issues/models.py

```diff
@@ -47,6 +49,10 @@ class Revision(PhabricatorModel):
     def __str__(self):
         return f"D{self.id} - {self.title}"
 
+    @property
+    def phabricator_url(self):
```

Comment:
Nit: maybe specify `phabricator_revision_url`

Reply:
My idea was to later add `phabricator_url` on the different model if needed (for Repo, Diff, mayber later Build, ..)